ecosystem/sep-0031.md

```diff
@@ -86,7 +86,7 @@ All endpoints respond with content type:
 1. The Sending Anchor makes [SEP-12 PUT /customer](sep-0012.md#customer-put) requests containing the collected information for each client.
 1. On successful registration (`202` HTTP status), the Sending Anchor makes a `POST /transactions` request to create a transaction record with the Receiving Anchor. This request contains the `id`s returned by the `PUT /customer` requests, the `transaction.fields` values collected from the Sending Client, as well as the transaction amount and asset information. The response will include a `id` that will be used to check the status of the transaction.
 1. The Sending Anchor makes `GET /transactions/:id` requests until the transaction's `status` is `pending_sender`. 
-1. The Sending Anchor submits the payment transaction to Stellar.
+1. The Sending Anchor submits the payment transaction to Stellar using the information provided in the `POST /transactions` response.
```

Comment:
❗ What information will it use? This is somewhat vague. If the intent is to say to send the an address that was specified in the post, it might be worth being more specific about that.

Reply:
The response includes the destination account and the memo to use in the transaction. I felt we should refer to the endpoint that describes those attributes instead of explicitly mentioning them here, which we don't do for the other calls described in this section.

Reply:
IMO if we describe all fields here the text tends to become outdated as new fields are included so it's easier to maintain this spec if we direct the user to the endpoint containing the desired information.

Reply:
Makes sense, thanks for explaining.